lib/ansible/errors/__init__.py

```diff
@@ -204,3 +204,10 @@ class AnsibleUndefinedVariable(AnsibleRuntimeError):
 class AnsibleFileNotFound(AnsibleRuntimeError):
     ''' a file missing failure '''
     pass
+
+class AnsibleActionSkip(AnsibleRuntimeError):
+    ''' an action runtime skip'''
+    pass
+class AnsibleActionFail(AnsibleRuntimeError):
+    ''' an action runtime failure'''
+    pass
```

Comment:
I think we should make a new hierarchy instead of inheriting from (eventually) AnsibleError.  These are actually for flow control rather than Errors.  For prior art, see how Python's exception hierarchy sets up StopIteration and GeneratorExit so that they do not inherit from StandardError.